public/index.html

```diff
@@ -15,6 +15,30 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <script>
```

Comment:
We should find another way for doing this.
the index.html should never be touched